packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -180,4 +156,49 @@ export class HttpServer extends RunnableModule {
 
     return serverClosePromise(this.server);
   }
+
+  private async initMetrics() {
+    const promRegistry = new Registry();
+    for (const service of this.#dependencies.services) {
```

Comment:
You're right @mkazlauskas, based on our program sharing a node between all registered services, the scaling of a backend to use multiple nodes is managed by running multiple instances of the provider server. The service-level metric is more suited to reporting the projected sync progress, which is what I had in mind when discussing the implementation with @iadmytro, which is the other metric we need here to visualise the health of the service.

Let's switch this to be reporting the projected sync status, then add another top level metric that reports the node sync status, which can be mapped to the first item in the services array.

Reply:
> Could you please attach some visuals of what is the metrics output and how it looks with the new sync status? 

![image](https://user-images.githubusercontent.com/95642896/225562048-d87fad11-bc05-4bb4-861f-1da2bb7af519.png)

Reply:
> ...then add another top level metric that reports the node sync status, which can be mapped to the first item in the services array.

It looks like the implementation has the **node sync percentage** being reported at the service level @iadmytro, rather than what I suggested above.

Reply:
'node sync percentage' moved to top level in [b806319](https://github.com/input-output-hk/cardano-js-sdk/pull/647/commits/b8063199a9caa511ed58b5aa65f578054be19be4)